docs/ruleset_functions/get_zones_hvac_system_humid_p.md

```diff
@@ -0,0 +1,41 @@
+## get_zones_hvac_system_humid_p
+
+**Description:** Get the list of P_RMR zones and corresponding HVAC system IDs where humidification is applicable.
+
+**Inputs:**
+- **P-RMR**: To determine if any zones and corresponding HVAC systems have humidification in the proposed.
+
+**Returns:**
+- **applicable_zones_hvac_sys_humid_dict_p**: A dictionary that saves all zones IDs as keys and a list of associated HVAC sys IDs as values that have humidification on the P_RMR.
+ 
+**Function Call:** None
+
+
+**Logic:**
+- For each zone in P_RMR: `zone_p in P_RMR...zones:`
+- Reset applicability flag: `rule_applicability_check = FALSE`  
+    - For each terminal unit associated with each zone: `terminal_p in zone_p:`
+        - Get the served_by_heating_ventilation_air_conditioning_systems for each terminal: `heating_ventilation_air_conditioning_systems_p = terminal_p.served_by_heating_ventilation_air_conditioning_systems`
+            - Create list of heating_ventilation_air_conditioning_systems_p add to as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_p = heating_ventilation_air_conditioning_systems_list_p.append(heating_ventilation_air_conditioning_systems_p)`                    
+    - Convert the list of heating_ventilation_air_conditioning_systems_list_p associated with the zone to a set and the back to a list to eliminate duplicates: `heating_ventilation_air_conditioning_systems_list_p = list(set(heating_ventilation_air_conditioning_systems_list_p))`
+     - For each hvac_p in heating_ventilation_air_conditioning_systems_list_p: `for hvac_p in heating_ventilation_air_conditioning_systems_list_p:`    
+        - Reset applicability flag_hvac: `rule_applicability_check_hvac = FALSE`  
+        - For each heating_system in hvac_p.heating_system: `for heating_system_p in hvac_p.heating_system:`
+            - Get humidification type: `humidification_type_p = heating_systems_p.humidification_type`
+            - Check if humidification type is adiabatic or other (if null or NONE then no humidification): `if humidification_type_p in ["ADIABATIC", "OTHER"]:`
+                - Set zone applicability flag: `rule_applicability_check = TRUE`
```

Comment:
@claperle can we rename this to 'humification_true' flag instead of 'rule applicability' flag? I'm assuming there will be two rules, one to check if baseline is modeled with humidication (if proposed has it) and one to check if the humidification type is adiabatic or non adiabatic. The rule applicability flag gets slightly confusing since we don't know all the rules this function could apply to - so clearing specifying that might make it clearer.

Reply:
Only applies to non-mandatory rules. Stay tuned for discussion on what exactly should be provided/checked for non-mandatory rules.

Reply:
I renamed it to "humidification_modeled_in_proposed_check"